File: t/CS/AnnotationsDemo/UI/UserInfoDialog.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace AnnotationsDemo
{
   public partial class UserInfoDialog : Form
   {
      public bool ShowNextTime
      {
         get
         {
            return !checkBox_DontShowAgain.Checked;
         }
      }

      private UserInfoDialog()
      {
         InitializeComponent();
      }

      public UserInfoDialog(string description)
      {
         InitializeComponent();
         _textboxDescription.Text = description;
      }
   }
}

File: t/VB/AnnotationsDemo/UI/UserInfoDialog.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Public Class UserInfoDialog
   Public Sub New(description As String)
      InitializeComponent()
      _textboxDescription.Text = description
   End Sub

   Public ReadOnly Property ShowNextTime() As Boolean
      Get
         Return Not checkBox_DontShowAgain.Checked
      End Get
   End Property
End Class

